<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFrameQueryData</class>
 <widget class="QFrame" name="QFrameQueryData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据查询</string>
  </property>
  <property name="whatsThis">
   <string>数据查询</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1151</width>
     <height>581</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabRecord">
    <attribute name="title">
     <string>车辆进出记录</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>691</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>691</width>
       <height>31</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>4</y>
        <width>671</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>车牌号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtPlate"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>起始时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dtRecordStart"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>截止时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dtRecordEnd"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnInOutRecordQuery">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>390</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutLeft"/>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>客户信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>车辆信息</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabCustomerInfo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
    </widget>
    <widget class="QTableView" name="tabVehicleInfo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>维修记录</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabServiceRecord">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>进出记录</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabInOutRecord">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>351</width>
       <height>302</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblInImage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>351</width>
       <height>302</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblOutImage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>390</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutRight"/>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutMain"/>
    </widget>
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWebView" name="webView">
     <property name="geometry">
      <rect>
       <x>669</x>
       <y>359</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>480</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>331</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lblTrend">
         <property name="text">
          <string>趋势图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbxType">
         <item>
          <property name="text">
           <string>按年</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按季</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按年周</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按年月周</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按日</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按时</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>图形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbxChartType"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
